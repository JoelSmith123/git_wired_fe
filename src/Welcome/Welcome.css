.Welcome {
  background-color: #4c4c4c;
  height: 800px;
  margin: 50px auto;
  width: 80%;
  /* align-content: space-around; */
  align-content: center;
  justify-content: space-around;
}

.welcome-content-description-title {
  color: #fff;
  text-align: center;
  padding-top: 50px;
}

.welcome-content-description {
  color: #fff;
  margin: 30px;
  width: 80%;
}

.welcome-content-instructions-title {
  color: #fff;
  text-align: center;
  padding-top: 100px;
}

.welcome-content-instructions {
  color: #fff;
  margin: 30px;
  width: 80%;
}
