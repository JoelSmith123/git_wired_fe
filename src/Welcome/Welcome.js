import React, { Component } from 'react'
import 'Welcome.css';

export default class Welcome extends Component {
  render() {
    return (
      <div className='Welcome'>
        <h2 className='welcome-content-description-title'>Description</h2>
        <p className='welcome-content-description'>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit,
          sed do eiusmod tempor incididunt ut labore et dolore magna
          aliqua. Ut enim ad minim veniam, quis nostrud exercitation
          ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
          aute irure dolor in reprehenderit in voluptate velit esse cillum
          dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
          non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
        <h2 className='welcome-content-instructions-title'>Get Started!</h2>
        <p className='welcome-content-instructions'>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit,
          sed do eiusmod tempor incididunt ut labore et dolore magna
          aliqua. Ut enim ad minim veniam, quis nostrud exercitation
          ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
          aute irure dolor in reprehenderit in voluptate velit esse cillum
          dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
          non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
      </div>
    )
  }
}