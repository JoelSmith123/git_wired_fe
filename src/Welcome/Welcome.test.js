import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount } from 'enzyme';
import Welcome from './Welcome.js'

describe('Welcome', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Welcome />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match snapshot', () => {
    const wrapper = shallow(<Welcome />)
    expect(wrapper).toMatchSnapshot()
  }) 

  xit('should render instructional content', () => {
    // TO DO - TEST ME
  })


});
