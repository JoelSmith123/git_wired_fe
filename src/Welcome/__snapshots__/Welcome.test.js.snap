// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="welcome-content-description-title"
        >
          Welcome to GitWired!
        </h2>,
        <div
          className="welcome-content-description"
        >
          <p>
            This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!
          </p>
          <p>
            This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.
          </p>
          <p>
            Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!
          </p>
        </div>,
        <h2
          className="welcome-content-instructions-title"
        >
          Get Started!
        </h2>,
        <div
          className="welcome-content-instructions"
        >
          <p>
            First - login with GitHub!
          </p>
          <p>
            Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!
          </p>
          <p>
            This app can be used with both your public & private repos, projects, and issues.
          </p>
          <p>
            Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!
          </p>
        </div>,
      ],
      "className": "Welcome",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to GitWired!",
          "className": "welcome-content-description-title",
        },
        "ref": null,
        "rendered": "Welcome to GitWired!",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!
            </p>,
            <p>
              This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.
            </p>,
            <p>
              Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!
            </p>,
          ],
          "className": "welcome-content-description",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!",
            },
            "ref": null,
            "rendered": "This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.",
            },
            "ref": null,
            "rendered": "This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!",
            },
            "ref": null,
            "rendered": "Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Get Started!",
          "className": "welcome-content-instructions-title",
        },
        "ref": null,
        "rendered": "Get Started!",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              First - login with GitHub!
            </p>,
            <p>
              Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!
            </p>,
            <p>
              This app can be used with both your public & private repos, projects, and issues.
            </p>,
            <p>
              Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!
            </p>,
          ],
          "className": "welcome-content-instructions",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "First - login with GitHub!",
            },
            "ref": null,
            "rendered": "First - login with GitHub!",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!",
            },
            "ref": null,
            "rendered": "Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This app can be used with both your public & private repos, projects, and issues.",
            },
            "ref": null,
            "rendered": "This app can be used with both your public & private repos, projects, and issues.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!",
            },
            "ref": null,
            "rendered": "Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="welcome-content-description-title"
          >
            Welcome to GitWired!
          </h2>,
          <div
            className="welcome-content-description"
          >
            <p>
              This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!
            </p>
            <p>
              This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.
            </p>
            <p>
              Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!
            </p>
          </div>,
          <h2
            className="welcome-content-instructions-title"
          >
            Get Started!
          </h2>,
          <div
            className="welcome-content-instructions"
          >
            <p>
              First - login with GitHub!
            </p>
            <p>
              Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!
            </p>
            <p>
              This app can be used with both your public & private repos, projects, and issues.
            </p>
            <p>
              Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!
            </p>
          </div>,
        ],
        "className": "Welcome",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to GitWired!",
            "className": "welcome-content-description-title",
          },
          "ref": null,
          "rendered": "Welcome to GitWired!",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!
              </p>,
              <p>
                This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.
              </p>,
              <p>
                Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!
              </p>,
            ],
            "className": "welcome-content-description",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!",
              },
              "ref": null,
              "rendered": "This is an organizational tool to setup minimal, static, black & white, sketch-like wireframes as a representation of your GitHub issue and project cards. By pairing wireframe elements to GitHub issues in a Repository & Project of your choosing, you can filter your features and GitHub cards based on completion status to see your feature progress in relation to all your features - the end goal!",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.",
              },
              "ref": null,
              "rendered": "This tool is intended for small teams, who desire simple and no-fuss wireframes where keeping track of feature completion is the highest priority in team communication.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!",
              },
              "ref": null,
              "rendered": "Leave the details in your user stories and UX PDFs - get a quick summary of the project with GitWired!",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Get Started!",
            "className": "welcome-content-instructions-title",
          },
          "ref": null,
          "rendered": "Get Started!",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                First - login with GitHub!
              </p>,
              <p>
                Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!
              </p>,
              <p>
                This app can be used with both your public & private repos, projects, and issues.
              </p>,
              <p>
                Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!
              </p>,
            ],
            "className": "welcome-content-instructions",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "First - login with GitHub!",
              },
              "ref": null,
              "rendered": "First - login with GitHub!",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!",
              },
              "ref": null,
              "rendered": "Everything in this app is directly tied to the current (and future) state of your GitHub account. And no personal user data is stored!",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This app can be used with both your public & private repos, projects, and issues.",
              },
              "ref": null,
              "rendered": "This app can be used with both your public & private repos, projects, and issues.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!",
              },
              "ref": null,
              "rendered": "Once you login, you can start a new project by selecting the collection of issues you'd like to work with and a template wireframe to attach them to. You're wireframe is already largely setup, with some customization flexibility, to get you up and running quickly!",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
